src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -1025,30 +1022,20 @@ namespace HVACSingleDuctInduc {
 
             // note we save the induced air flow for use by the hw and cw coil sizing routines
-            termUnitSizing.AirVolFlow = state.dataHVACSingleDuctInduc->IndUnit(IUNum).MaxTotAirVolFlow *
-                                        state.dataHVACSingleDuctInduc->IndUnit(IUNum).InducRatio /
-                                        (1.0 + state.dataHVACSingleDuctInduc->IndUnit(IUNum).InducRatio);
+            termUnitSizing.AirVolFlow = indUnit.MaxTotAirVolFlow *
+                                        indUnit.InducRatio /
+                                        (1.0 + indUnit.InducRatio);
             // save the max hot and cold water flows for use in coil sizing
-            termUnitSizing.MaxHWVolFlow = state.dataHVACSingleDuctInduc->IndUnit(IUNum).MaxVolHotWaterFlow;
-            termUnitSizing.MaxCWVolFlow = state.dataHVACSingleDuctInduc->IndUnit(IUNum).MaxVolColdWaterFlow;
+            termUnitSizing.MaxHWVolFlow = indUnit.MaxVolHotWaterFlow;
+            termUnitSizing.MaxCWVolFlow = indUnit.MaxVolColdWaterFlow;
             // save the design load used for reporting
-            termUnitSizing.DesCoolingLoad = state.dataHVACSingleDuctInduc->IndUnit(IUNum).DesCoolingLoad;
-            termUnitSizing.DesHeatingLoad = state.dataHVACSingleDuctInduc->IndUnit(IUNum).DesHeatingLoad;
+            termUnitSizing.DesCoolingLoad = indUnit.DesCoolingLoad;
+            termUnitSizing.DesHeatingLoad = indUnit.DesHeatingLoad;
             // save the induction ratio for use in subsequent sizing calcs
-            termUnitSizing.InducRat = state.dataHVACSingleDuctInduc->IndUnit(IUNum).InducRatio;
-            if (Util::SameString(state.dataHVACSingleDuctInduc->IndUnit(IUNum).HCoilType, "Coil:Heating:Water")) {
-                WaterCoils::SetCoilDesFlow(state,
-                                           state.dataHVACSingleDuctInduc->IndUnit(IUNum).HCoilType,
-                                           state.dataHVACSingleDuctInduc->IndUnit(IUNum).HCoil,
-                                           termUnitSizing.AirVolFlow,
-                                           ErrorsFound);
-            }
-            if (Util::SameString(state.dataHVACSingleDuctInduc->IndUnit(IUNum).CCoilType, "Coil:Cooling:Water:DetailedGeometry")) {
-                WaterCoils::SetCoilDesFlow(state,
-                                           state.dataHVACSingleDuctInduc->IndUnit(IUNum).CCoilType,
-                                           state.dataHVACSingleDuctInduc->IndUnit(IUNum).CCoil,
-                                           termUnitSizing.AirVolFlow,
-                                           ErrorsFound);
+            termUnitSizing.InducRat = indUnit.InducRatio;
+            if (indUnit.heatCoilType == HVAC::CoilType::CoolingWater ||
+                indUnit.heatCoilType == HVAC::CoilType::CoolingWaterDetailed) {
+                WaterCoils::SetCoilDesFlow(state, indUnit.heatCoilNum, termUnitSizing.AirVolFlow);
```

Comment:
The diffs with 5ZoneFPIU is here. SetCoilDesFlow was called for heating and cooling coils. There branch munged that call.